controllers/feeds/getAnonymousFeedChat.js

```diff
@@ -58,7 +58,7 @@ const getAnonymousFeedChatService = async (req, res) => {
         actor,
         isMediaOnlyMessage = false,
         showToast = false
-      } = getDetail(req, b, mySignedId, req?.userId);
+      } = getDetail(req, b, mySignedId, req?.userId, options);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `options` parameter is now passed to the `getDetail` function. This change seems fine as long as the `getDetail` function properly handles this new parameter. However, it would be better if there was some validation or error handling for the `options` parameter before passing it to the `getDetail` function.
```diff
- } = getDetail(req, b, mySignedId, req?.userId, options);
+ const validOptions = validateOptions(options); // validateOptions is a hypothetical function that validates the options object
+ } = getDetail(req, b, mySignedId, req?.userId, validOptions);
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->